.gitmodules

```diff
@@ -13,8 +19,8 @@
         fxDONOTUSEurl = https://github.com/MPAS-Dev/MPAS-Model.git
 [submodule "ncar-physics"]
         path          = src/physics/ncar_ccpp
-        url           = https://github.com/ESCOMP/atmospheric_physics
-        fxtag         = 098585940ad763be58ebab849bb8eaf325fda42a
+	url           = https://github.com/peverwhee/atmospheric_physics
+	fxtag         = 61bd9d3dac2abb11bb1e44a2ca34b401da0f44b1
```

Comment:
This also needs to be updated

Reply:
thanks for the reminder - updated to point to the latest (including the diagnostic schemes) development hash of ESCOMP/atmospheric_physics